raiden-ts/tests/unit/fixtures.ts

```diff
@@ -166,6 +166,22 @@ export const secret = makeSecret();
 export const secrethash = getSecrethash(secret);
 export const amount = bigNumberify(10) as UInt<32>;
 export const fee = bigNumberify(3) as Int<32>;
+export const pfsAddress = makeAddress();
+
+/**
+ * @param raiden - Instance of MockedRaiden
+ * @returns Mocked IOU type Object
+ */
+export function makeIou(raiden: MockedRaiden): IOU {
```

Comment:
This function is only relevant for this specific test suite. Please, move it to the top of the `path.spec.ts` file, no need for it to be in the shared fixtures.

Reply:
ok sure will move it at the top of this test suite